Clarify use of Repair/Medkits to Stabilize

Turtlelord26 opened the pull request:
#506
Now provides +20 to the check. Also did some clarity work on the Stabilization section.

Turtlelord26 committed:
Clarify use of Repair/Medkits to Stabilize

#506
Now provides +20 to the check. Also did some clarity work on the Stabilization section.

rosebrallier commented on 01_02_Combat_Rules.txt:
```diff
@@ -888,10 +888,16 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
-bonuses from medical equipment such as medic-kits are added to the roll. Once stabilized,
-you may take actions  and speak as normal.
+	As an action, a character may choose an adjacent, Downed character and 
+attempt to Stabilize them. If successful on a DC 80 Medicine (Dex usually) check, 
```
"usually Dex" reason: formal tone

rosebrallier commented on 01_02_Combat_Rules.txt:
```diff
@@ -888,10 +888,16 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
+attempt to Stabilize them. If successful on a DC 80 Medicine (Dex usually) check, 
+the character now has 1 HP, is no longer Downed, and may take Actions as normal 
+within the initiative order. Note that most persistent damage effects, like 
+bleed or burn tokens, fall off of characters when they are Downed.
```
Are we consistently calling them "tokens"?

rosebrallier reviewed (changes requested):
When we stabilize a mechanical target, should it be Medicine or Repair check?
And what happens on a failed check?

Turtlelord26 committed:
Update Stabilization Rules per Review

Added failure, critical success & failure, repair vs medicine, and repair/medkit persistence.
Added Blee's houserule for maximum number of revivals per encounter.

bleehu commented on 01_02_Combat_Rules.txt:
```diff
@@ -888,12 +888,31 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
-you may take actions  and speak as normal.
-
-	Don't forget that a recently stabilized character may need  to take actions to
+	As an action, a character may choose an adjacent, Downed character and 
```
Nonblocking; I think we should lengthen the time it takes to stabilize a character so that it's longer than the time to raise and fire a weapon. Perhaps I should also lengthen the time for the stabilize Doctor Procedure.

Turtlelord26 replied:
I concur, but if we change one we should change the other too, and that makes it scope creep I think?
If we decide on longer than 2 actions, we can also delete the bit I put in here about reusing repair/medkits.

bleehu commented on 01_02_Combat_Rules.txt:
```diff
@@ -888,12 +888,31 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
+against their revivals per encounter limit. On critical failure (01-05 before 
+modifiers), the bleeding (or equivalent) gets worse: the target must immediately 
+roll an additional Death Save. If the total check is then still high enough to 
+succeed and the target is still alive, they are then Stabilized as normal.
```
I love the crit-success/fail detail! Fantastic!

bleehu commented on 01_02_Combat_Rules.txt:
```diff
@@ -888,12 +888,31 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
+character may expend one Medkit (for primarily organic targets) or one Repair Kit 
+(for primarily mechanical targets) to grant +20 to the Medicine check. Feel free 
+to scavenge one from the target, they probably won't make a fuss. One kit provides 
+a bonus to all Stabilization attempts made on the same target in one round.
```
I also really really like how you called out assistance as a specific mechanic! Great work!

Turtlelord26 replied:
Though I didn't mention what it does. We'll need a general rule elsewhere. *New Issue

bleehu reviewed (approved):
One nonblocking issue, but we can handle that elsewhere.

rosebrallier commented on 01_02_Combat_Rules.txt:
```diff
@@ -888,12 +888,31 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
+fall off of characters when they are Downed.
+
+	If a character fails a stabilization check, nothing happens. It's just 
+taking longer to stem the bleeding. On critical success (91-100 before modifiers), 
```
Did we kill Luck-based crit-range modification?

Turtlelord26 replied:
Negative, will fix. Note that the fix will reference currently nonexistent rules re critical success on skill checks (issued elsewhere).

rosebrallier commented on 01_02_Combat_Rules.txt:
```diff
@@ -888,12 +888,31 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
+your excellent care allows you to choose whether the target counts this revival 
+against their revivals per encounter limit. On critical failure (01-05 before 
+modifiers), the bleeding (or equivalent) gets worse: the target must immediately 
+roll an additional Death Save. If the total check is then still high enough to 
```
It's a little confusing to me that you could succeed at a task when you critically failed the roll.

Turtlelord26 replied:
I was thinking success-at-a-cost style for this action, because I didn't think I would like ever auto-failing a non-Doctor serious enough to put 50+ skill points into Medicine (/non-Engineer with 50 points in Repair). Not unwilling to revert.

Turtlelord26 replied:
@bleehu your thoughts on this also solicited

rosebrallier reviewed (commented):
So much detail and specificity! So much good! I do have some concerns about critical success and failure.

Turtlelord26 committed:
Critical Success now (probably) supports Luck
